packages/open-next/src/utils/lru.ts

```diff
@@ -0,0 +1,28 @@
+export class LRUCache<T> {
+  private cache: Map<string, T> = new Map();
+
+  constructor(private maxSize: number) {}
+
+  get(key: string) {
+    const result = this.cache.get(key);
```

Comment:
`.has` would be allow storing `null` but as we don't need that, maybe only add a comment on the class?